shared/config/init.vendor.rc

```diff
@@ -0,0 +1,152 @@
+on early-init
```

Comment:
Hi @arminn,

I've checked why this change was done. It was introduced in order to prevent the renaming of the eth0 interface during the startup of the system. I've extended the commit message.